{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_drf_views.py

```diff
@@ -31,3 +32,10 @@ def test_me(self, user: User, rf: RequestFactory):
             "name": user.name,
             "url": f"http://testserver/api/users/{user.username}/",
         }
+
+
+class TestAPISchemaView:
+    def test_api_schema_generated_successfully(self, admin_client):
+        url = reverse("api-schema")
+        response = admin_client.get(url)
+        assert response.status_code == 200
```

Comment:
If we only have one method, we can keep it simple with a plain function:

```suggestion
def test_api_schema_generated_successfully(admin_client):
    url = reverse("api-schema")
    response = admin_client.get(url)
    assert response.status_code == 200
```

Also, it sounds like it would be better located [with the other swagger tests `users/tests/test_swagger_ui.py`](https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/users/tests/test_swagger_ui.py)

Since this will test both the UI and the schema, we could rename it to `test_swagger.py`.